src/bootstrap/native.rs

```diff
@@ -181,7 +181,7 @@ impl Step for Llvm {
             .define("LLVM_TARGET_ARCH", target_native.split('-').next().unwrap())
             .define("LLVM_DEFAULT_TARGET_TRIPLE", target_native);
 
-        if target != "aarch64-apple-darwin" {
+        if target != "aarch64-apple-darwin" && !target.contains("windows") {
```

Comment:
this will also match other windows targets, for example `x86_64-pc-windows-msvc`

Reply:
Exactly, currently `windows-msvc` doesn't link Zlib at all but who knows what is going to change within GitHub Actions CI...